package.json

```diff
@@ -9,7 +9,7 @@
     "build": "node-gyp build",
     "test": "make test",
     "install": "node-gyp rebuild",
-    "prepack": "node ./ci/prepublish.js"
+    "_prepack": "node ./ci/prepublish.js"
```

Comment:
This was messing with package version while built in a brach, shoudl remove `_` before merging back to master.